templates/js/www

```diff
@@ -58,5 +58,5 @@ function onError(error) {
  */
 
 function onListening() {
-  debug('Listening on port ' + server.address().port);
+  debug('Listening on port ' + (server.address().port || server.address()));
```

Comment:
yes, it prints Listening on port \.\pipe\4323f7c5 in case of iisnode and it prints the integer port if the port was set to integer.